### ISSUE TITLE
Java Deprecations

### ISSUE THREAD

**Issue by cdietrich** [eclipse/Xpect#374]
Description	Resource	Path	Location	Type
The constructor URL(String) is deprecated since version 20	FileExtensionInfoTest.java	/org.eclipse.xpect.tests/src/org/eclipse/xpect/tests	line 205	Java Problem

(also at a 2nd place)

**Comment by Nishantware** [comment]
can you send location path one more time (path only) ?

**Comment by cdietrich** [comment]
FileExtensionInfoTest.java	/org.eclipse.xpect.tests/src/org/eclipse/xpect/tests	line 205	
JavaProject.java	/org.eclipse.xpect.xtext.lib/src/org/eclipse/xpect/xtext/lib/setup/workspace	line 79

**Comment by Nishantware** [comment]
from description  i didn't get nothing can you share the screenshot what is the problem?

**Comment by cdietrich** [comment]
<img width="812" alt="Image" src="https://github.com/user-attachments/assets/3b96ee1c-58b8-4eb7-9729-d16dc738f76a" />

<img width="847" alt="Image" src="https://github.com/user-attachments/assets/8bcbb22a-5ebd-46e7-aea2-be9bc9b3ce77" />

**Comment by Nishantware** [comment]
@cdietrich yup i understood the issue so what type of changes need to i will do that easily .

**Comment by cdietrich** [comment]
if i knew what exactly to replace it with it would have been done 2 or 3 months ago

**Comment by Nishantware** [comment]
yes you are taking also right but you didn't provide what type of solution you want for that reason the issue still have not solve yet .
if we don't know what is the solution for that how can anyone solve these problem am i right or wrong ?

**Comment by cdietrich** [comment]
read the javadoc. read the internet. find out what the solution is
(as you would do for each deprecation in the world)
